### DISCUSSION TITLE
WebUSB bScheme set to 0xff , chrome do not show notification

### DISCUSSION THREAD

**Discussion by Jacques-Zhao** [hathach/tinyusb#1390] [Q&A] [answered]
Hi 
I am using Webusb to develop my new project and my ESP32-S3 device cannot access the Internet at this time. I want landing Page to point to a local HTML file that download from [webusb-serial](https://example.tinyusb.org/webusb-serial/) , so I try to set bscheme to 0xFF according to the wicg [document](https://wicg.github.io/webusb/#url-descriptor).
Then set the URL to file:///path_to_file.html.
Unfortunately, Chrome will not show a notification when the USB device is plugged in, even the URL prefix change to  HTTPS .

If I change bScheme to 1 or 0(according to the [policy and mechanism](https://www.chromium.org/Home/chromium-security/deprecating-powerful-features-on-insecure-origins/)), Chrome will show notifications normally.

**Comment by Jacques-Zhao** [comment, marked as answer]
I also writed an issue in webusb project, and they confirmed that this parameter is not currently supported by Chrome. https://github.com/WICG/webusb/issues/216

**Reply by joeahn95** [reply]
Is it possible to take a look at your WebUSB project? I am currently trying to create a simple application using Webusb and ESP32-S3, but I cannot find any good examples. Thanks.
